MIP61/MIP61.md

```diff
@@ -128,11 +136,13 @@ The parameters that can be modified are:
 - Exponential factor
 - Maximum annual compensation per delegate
 
-Recognized Delegates are potentially conflicted in consideration of these proposals. There may be a temptation to approve increases in compensation and vote against decreases. Hence, a Signal Request is required to assess broader community sentiment and spotlight any positions on the MIP61c5 proposal staked out by Recognized Delegates. [Sunshine is the best disinfectant](https://quoteinvestigator.com/2020/09/22/sunlight/). The Signal Request Poll must be posted at the start of the Frozen Period.
+Recognized Delegates are potentially conflicted in consideration of these proposals. There may be a temptation to approve increases in compensation and vote against decreases. Hence, a Signal Request is required to assess broader community sentiment and spotlight any positions on the MIP61c5 proposal staked out by Recognized Delegates. [Sunshine is the best disinfectant.](https://quoteinvestigator.com/2020/09/22/sunlight/) The Signal Request Poll must be posted at the start of the Frozen Period.
```

Comment:
Would just leave out the quote and link.

Reply:
Done (see below)